src/code/utils/translate.coffee

```diff
@@ -5,7 +5,8 @@ varRegExp = /%\{\s*([^}\s]*)\s*\}/g
 
 translate = (key, vars={}, lang=defaultLang) ->
   translation = translations[lang]?[key] or key
-  translation.replace varRegExp, (match, key) ->
-    if vars.hasOwnProperty key then vars[key] else "'** UKNOWN KEY: #{key} **"
+  if translation
```

Comment:
Translate can only be falsy if key itself is falsy. Is that likely? We're always passing something into `tr`.